src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,33 @@
+package study;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String data){
+       if(data == null || data.isEmpty()) return 0;
```

Comment:
- IDE로 IntelliJ IDEA를 사용 중이시라면, 자동 포맷팅(`Ctrl + Alt + L`) 기능을 활용해보시는 건 어떨까요?
- if문 안의 코드가 한 줄이더라도 중괄호를 추가해보면 좋을 것 같아요.
  중괄호를 추가함으로써 얻을 수 있는 이점은 어떤 게 있을까요?
```suggestion
        if (data == null || data.isEmpty()) {
            return 0;
        }
```